### PR TITLE
feat-scss

### PR DESCRIPTION
**Pull request by jazzcodes**
Configured SCSS

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "extends": ["airbnb-base", "prettier"],
+    "plugins": ["prettier"],
+    "env": {
+      "node": true,
+      "es6": true,
+      "browser": true
+    },
+    "rules": {
+      "no-console": "off"
```
Comment (mandeeps708) [inline .eslintrc.json]: Let's turn it on.

File: css/styles.css [hunk 2]
```diff
@@ -1,189 +1,177 @@
...
-    --secondary-400:#596780;
-}
-
-
-html{  
-    font-size: 62.5%;
-    height: 100vh;
-    /* display: flex;
+*, *::after, *::before {
+  box-sizing: border-box; }
```
Comment (mandeeps708) [inline css/styles.css:2]: The closing brace should be in new line. 

Are you following some standard for this?

But if you have shifted to scss, then we can skip making changes to this.

**Review by mandeeps708 — CHANGES_REQUESTED** [review]

**Comment by mandeeps708** [comment]
@jazzcodes Can you close this PR as #5 has all this code.

**Comment by jazzcodes** [comment]
PR #5 has all this code.

**Pull request closed without merging**